src/vdom.rs

```diff
@@ -195,48 +195,43 @@ impl<Ms, Mdl, ElC: View<Ms> + 'static, GMs: 'static> App<Ms, Mdl, ElC, GMs> {
         }
     }
 
-    /// App initialization: Collect its fundamental components, setup, and perform
-    /// an initial render.
-    pub fn run(self) -> Self {
-        self.process_cmd_and_msg_queue(
-            self.cfg
-                .initial_orders
-                .replace(None)
-                .expect("initial_orders should be set in AppBuilder::finish")
-                .effects,
-        );
-        // Our initial render. Can't initialize in new due to mailbox() requiring self.
+    fn bootstrapped_vdom(&self) -> El<Ms> {
```

Comment:
Can we rename it to `bootstrap_vdom`? I would assume that `bootstrapped_vdom` is only a getter.

Reply:
Sure.